.gitignore

```diff
@@ -2,3 +2,5 @@ deploy-to-wp-org.sh
 node_modules
 Gruntfile.js
 package.json
+.idea
+*.iml
```

Comment:
Were these added intentionally?

Reply:
Yep, those are used by WebStorm. We have the same ignores in `woocommerce-services`, it's unrelated with this change but it doesn't hurt :)